homeassistant/components/zwave_me/number.py

```diff
@@ -0,0 +1,45 @@
+"""Representation of a switchMultilevel."""
```

Comment:
Please exclude non tested modules from coverage calculation in `.coveragerc`. The config flow must be tested 100% though.